content/applications/finance/payment_providers/paypal.rst

```diff
@@ -2,120 +2,78 @@
 PayPal
 ======
 
-`Paypal <https://www.paypal.com/>`_ is an American online payment provider available worldwide, and
+`Paypal <https://www.paypal.com/>`_ is an American online payment provider available worldwide and
 one of the few that does not charge a subscription fee.
 
 .. note::
    While PayPal is available in `over 200 countries/regions
    <https://www.paypal.com/webapps/mpp/country-worldwide>`_, only `a selection of currencies are
    supported <https://developer.paypal.com/docs/reports/reference/paypal-supported-currencies>`_.
 
-Settings in PayPal
-==================
+.. _payment_providers/paypal/configuration-paypal:
 
-To access your PayPal account settings, log into PayPal, open the :guilabel:`Account Settings`, and
-open the :guilabel:`Website payments` menu.
+Configuration in PayPal
+=======================
 
-.. image:: paypal/paypal-account.png
-   :align: center
-   :alt: PayPal account menu
+#. `Log into your PayPal Developer Dashboard <https://developer.paypal.com/dashboard/>`_.
+#. Click :guilabel:`Apps & Credentials` and click :guilabel:`Create App`.
+#. Enter an :guilabel:`App Name` and click :guilabel:`Create App`.
+#. Copy the :guilabel:`Client ID` and :guilabel:`Secret` and save them for
+   :ref:`later <payment_providers/paypal/configuration-odoo>`.
 
 .. important::
-   Note that for PayPal to work **in Odoo**, the options :ref:`Auto Return <paypal/auto-return>` and
-   :ref:`PDT <paypal/pdt>` **must** be enabled.
+   If you use customer names or addresses that include accented or non-Latin characters, you
+   **must** configure the encoding format of the payment request sent by Odoo to PayPal to avoid
+   transaction failures without notice. To do so, access the `PayPal button language encoding
+   setting <https://www.paypal.com/cgi-bin/websrc?cmd=_profile-language-encoding>`_, click
+   :guilabel:`More Options`, and set the :guilabel:`Encoding` field to :guilabel:`UTF-8`.
```

Comment:
I don't know if that's still true with the current API; we'll test it and get back to you here :)

Reply:
This is actually very hard to test so let's simply keep this notice. It's probably beneficial anyway :)